docs/settingup.md

```diff
@@ -1114,7 +1114,7 @@ Unpublish a bookmark.
 
 ### Example
 
-Unpublish bookmark previously created in the script with an id `bookmark1`.
+Unpublish the bookmark with id "bookmark1" that was created earlier on in the script.
```

Comment:
Could you please "highlight" id (`id`), so that it corresponds to the way it is presented under Settings?